applications/app/controllers/GalleryController.scala

```diff
@@ -56,7 +54,7 @@ object GalleryController extends Controller with RendersItemResponse with Loggin
     val htmlResponse: (() => Html) = () =>
       views.html.gallery(model, model.related, model.index)
     val jsonResponse = () =>
-      views.html.fragments.galleryBody(model.gallery, model.related, model.index)
+      views.html.fragments.galleryBody(model.gallery, model, model.related, model.index)
```

Comment:
Can we just pass the `model` through here? Is `galleryBody` ever called where these attributes don't come from the same model?